pom.xml

```
@@ -55,7 +55,7 @@
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
 
         <!-- Build properties -->
-        <cs.jdk.version>11</cs.jdk.version>
+        <cs.jdk.version>17</cs.jdk.version>
```

if we do 21 we have more time to think of our sins.
```suggestion
        <cs.jdk.version>21</cs.jdk.version>
```

---

so this remains to be seen. depending on availabilities of libraries